test/src/androidTest/kotlin/io/realm/shared/RealmTests.kt

```diff
@@ -348,4 +353,23 @@ class RealmTests {
         // some way (or the writer to be backed by another thread).
         assertEquals(1, realm.objects(Parent::class).size)
     }
+
+    @Test
+    @Suppress("invisible_member")
+    fun writesOnFrozenRealm() {
+        val dispatcher = newSingleThreadContext("background")
+        runBlocking {
+            realm.write {
+                copyToRealm(Parent())
+            }
+        }
+        runBlocking(dispatcher) {
+            assertTrue { realm.isFrozen }
```

Comment:
I didn't thought that `isFrozen` was available on the JVM?

Reply:
No, just some quick assert for local native testing. Deleted it again.